internal/testutil/treedump.go

```diff
@@ -74,7 +74,12 @@ func TreeDumpEntry(entry *fsutil.Entry) string {
 		return fmt.Sprintf("dir %#o", fperm)
 	case fs.ModeSymlink:
 		return fmt.Sprintf("symlink %s", entry.Link)
-	case 0: // Regular
+	case 0:
+		// Hard link.
+		if entry.Link != "" {
+			return fmt.Sprintf("hard link %#o %s", fperm, entry.Link)
```

Comment:
Let's use the exact same format that we use in `TreeDumpReport`.

Reply:
Updated in [4313070](https://github.com/letFunny/chisel/pull/17/commits/4313070b5a0af67b4c1cdca323c0eb03d7f882b9).